### DISCUSSION TITLE
Добавить к дате только рабочие дни

### DISCUSSION THREAD

**Discussion by fastorium** [totumonline/totum-ru-issues-and-discussions#406] [Как это сделать?] [answered]
Здравствуйте, как в totem коде к дате добавить только рабочие дни?
Чисто на php нагуглил такую функцию: date('d/m/Y',strtotime("+10 weekdays"));

И еще в документации нет описания функции "dateWeekDay", на которую ссылается текст из форматирования даты.

**Comment by erpic-ru** [comment, marked as answer]
Пару лет считаю рабочие дни запросами к спецсервису. 

```
= : getFromScript(uri: $link)
     ~link: strAdd(str: "https://isdayoff.ru/"; str: #date)
```
Можно загрузить и сохранить на пару лет вперед, а дальше, например, выполнять selectlist по списку от нужной даты, до всех значений с единичками с лимитом в нужное число дней и брать последний элемент списка

[Ещё тут обсуждалось](https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/62)
